tests/foreman/cli/test_remoteexecution.py

```diff
@@ -252,7 +252,12 @@ def test_positive_run_default_job_template(self):
             'inputs': 'command="ls"',
             'search-query': "name ~ {0}".format(self.client.hostname),
         })
-        self.assertEqual(invocation_command['success'], u'1')
+        self.assertEqual(invocation_command['success'], u'1',
```

Comment:
Please, provide an example of `get_output` command result and `host output: {0}'.format(' '.join(get_output...)`

Anyway, suggest to use whether:
```
self.assertEqual(
            invocation_command['success'], 
            u'1', 
            'host output: {0}'.format(
                ' '.join(JobInvocation.get_output({
                    'id': invocation_command[u'id'], 
                    'host': self.client.hostname})
                )
            )
        )
```
or
```
        job_output = JobInvocation.get_output({
            'id': invocation_command[u'id'],
            'host': self.client.hostname
        })
        self.assertEqual(
            invocation_command['success'],
            u'1',
            'host output: {0}'.format(' '.join(job_output))
        )
```

But first is preferable.